Server/VikingVault.Services/PDFGeneratorService.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.Services.Abstractions;
+using Syncfusion.Pdf;
+using Syncfusion.Pdf.Graphics;
+using Syncfusion.Drawing;
+using System.IO;
+using Microsoft.AspNetCore.Mvc;
+using Syncfusion.Pdf.Tables;
+using System.Data;
+using Syncfusion.Pdf.Grid;
+using VikingVault.DataAccess.Models;
+
+namespace VikingVault.Services
+{
+    public class PDFGeneratorService : IPDFGeneratorService
+    {
+        private ITransactionService _transactionService;
+
+        public PDFGeneratorService(ITransactionService transactionService)
+        {
+            _transactionService = transactionService;
+        }
+
+        public string GetTimeFilterHeaderText(string timeFilter)
+        {
+            if (timeFilter == "day")
```

Comment:
Please make the code blocks consistent to the .NET styling (each block on their own line)
{
}

Reply:
Yup, done.